src/common/multirow.js

```diff
@@ -1,12 +1,11 @@
-import logo from '../images.png';
-import Accordion from '../component/searchresult/accordian';
-import Multisingle from '../common/multisingle';
-const Multirow=(props)=>{ 
-    return (
-        <Accordion props={props}  sdata={props.sdata}>
-        <Multisingle flight={props.data}/>
-        <Multisingle flight={props.sdata}/>
-        </Accordion>
-      );
-  }
-  export default Multirow; 
\ No newline at end of file
+import Accordion from "../component/searchresult/accordian";
+import Multisingle from "../common/multisingle";
+const Multirow = (props) => {
+  return (
+    <Accordion props={props} sdata={props.sdata}>
```

Comment:
what is sdata?

Reply:
sdata is data for second flights data in multiflight

Reply:
please use good variable name